pages/index.tsx

```diff
@@ -75,679 +21,7 @@ const HomePage: ThirdwebNextPage = () => {
-                </ListItem>
-                <ListItem>
-                  <ListIcon as={FiCheck} color="green.500" />
-                  Zero fees on transactions
-                </ListItem>
-                <ListItem>
-                  <ListIcon as={FiCheck} color="green.500" />
-                  New features added every week
-                </ListItem>
-                <ListItem>
-                  <ListIcon as={FiCheck} color="green.500" />
-                  Save on gas fees with advanced optimizations
-                </ListItem>
-              </List>
-              <LightMode>
-                <GeneralCta title="Start building today" size="lg" w="100%" />
-              </LightMode>
-            </Box>
-          </SimpleGrid>
-        </HomepageSection>
-
-        <HomepageSection id="examples" middleGradient>
-          <Flex
-            flexDir="column"
-            py={{ base: 12, lg: 24 }}
-            align="center"
-            gap={{ base: 12, lg: 24 }}
-          >
-            <Heading
-              as="h2"
-              bgGradient="linear(to-r, #907EFF, #C5D8FF)"
-              bgClip="text"
-              size="display.md"
-              textAlign="center"
-            >
-              Build anything.
-            </Heading>
-            <ExamplesSection />
-          </Flex>
-        </HomepageSection>
-        <HomepageSection id="get-started" bottomPattern>
-          <Flex
-            flexDir="column"
-            pt={{ base: 12, lg: 24 }}
-            pb={{ base: 24, lg: 0 }}
-            align="center"
-            gap={{ base: 6, md: 8 }}
-          >
-            <Center mb={6} pt={{ base: 8, lg: 24 }}>
-              <Center p={2} position="relative" mb={6}>
-                <Box
-                  position="absolute"
-                  bgGradient="linear(to-r, #F213A4, #040BBF)"
-                  top={0}
-                  left={0}
-                  bottom={0}
-                  right={0}
-                  borderRadius="3xl"
-                  overflow="visible"
-                  filter="blur(15px)"
-                />
-
-                <ChakraNextImage
-                  alt=""
-                  boxSize={{ base: 24, md: 32 }}
-                  placeholder="empty"
-                  src={WhiteLogo}
-                />
-              </Center>
-            </Center>
-            <Heading as="h2" size="display.md" textAlign="center">
-              Get started with thirdweb
-            </Heading>
-            <Heading as="h3" maxW="600px" textAlign="center" size="subtitle.lg">
-              Build web3 apps with ease. Get instant access.
-            </Heading>
-            <LightMode>
-              <GeneralCta
-                title="Start building for free"
-                size="lg"
-                w={{ base: "full", md: "inherit" }}
-              />
-            </LightMode>
-          </Flex>
-        </HomepageSection>
-
+        <HomePageSections />
```

Comment:
I think I would still like the sections to be here in-line, we can do the dynamic import in this file, too.

Reason being as we add and remove sections / move them around it's not an extra step to first go to the sections file (that doesn't really do anything but to render them anyways)

Reply:
@jnsdls Ok, Let me Delete `<HomePageSections/>` and move all sections to `pages/index.tsx`